SingularityUI/app/components/logs/FileNotFound.jsx

```diff
@@ -1,8 +1,33 @@
-import React from 'react';
+import React, {PropTypes} from 'react';
```

Comment:
I've been doing spaces in this one, we don't enforce that with linting, but I think it looks nicer here.